arch/x86_64/mcount-dynamic.c

```diff
@@ -14,10 +14,11 @@
 #define PAGE_SIZE  4096
 #define XRAY_SECT  "xray_instr_map"
 #define MCOUNTLOC_SECT  "__mcount_loc"
-
-#define CALL_INSN_SIZE  5
 #define JMP_INSN_SIZE   6
 
+unsigned char call_insn[] = { 0xe8, 0x00, 0x00, 0x00, 0x00 };
+unsigned int INSTRUMENT_CODESIZE = sizeof(call_insn);
```

Comment:
We don't use upper case letters for variables.  Please make it a macro or use lower case.

Also I think `CALL_INSN_SIZE` makes more sense for some cases at least.

Reply:
this code has changed to follow your advice.